libs/agno/agno/team/team.py

```diff
@@ -2358,8 +2358,13 @@ def _build_reasoning_step_panel(
 
     def _get_member_name(self, agent_id: str) -> str:
         for member in self.members:
-            if member.agent_id == agent_id:
-                return member.name
+            if isinstance(member, Agent):
+                if member.agent_id and member.agent_id == agent_id:
+                    return member.name
+            elif isinstance(member, Team):
+                for sub_member in member.members:
+                    if sub_member.agent_id and sub_member.agent_id == agent_id:
+                        return sub_member.name
```

Comment:
Sure about this? I think the agent_id would be from the response on the team response's member_responses, which should itself be a TeamResponse, which would have a `team_id`?